### PR TITLE
Update to version 1.0.4

### PR DESCRIPTION
**Pull request by ivanholmes**

### CODE REVIEW

File: Library/Formula/libtorrent-rasterbar.rb [hunk 1]
```diff
@@ -13,10 +13,6 @@ class LibtorrentRasterbar < Formula
   end
 
   bottle do
-    cellar :any
```
Comment (DomT4) [inline Library/Formula/libtorrent-rasterbar.rb]: You can leave this block. The bot handles it.

Comment (ivanholmes) [inline Library/Formula/libtorrent-rasterbar.rb]: Alright, thanks. It's the first time I've done this so it's good to know :)

Comment (DomT4) [inline Library/Formula/libtorrent-rasterbar.rb]: No worries :). Essentially the test-bot we have in place creates new bottles for the formula, and then when a maintainer pulls the update with ` brew pull --bottle ` the bottles are published on Bintray and existing bottle block is replaced with the new block.

File: Library/Formula/libtorrent-rasterbar.rb [hunk 2]
```diff
@@ -2,8 +2,8 @@
 
 class LibtorrentRasterbar < Formula
   homepage "http://sourceforge.net/projects/libtorrent/"
-  url "https://downloads.sourceforge.net/project/libtorrent/libtorrent/libtorrent-rasterbar-1.0.3.tar.gz"
-  sha1 "ccdd8bdba178b300921b15b18dfe8c0705f7eb07"
+  url "https://downloads.sourceforge.net/project/libtorrent/libtorrent/libtorrent-rasterbar-1.0.4.tar.gz"
+  sha1 "a2029f10e7ae9f8f1885c6cd308b737482c1d737"
```
Comment (DomT4) [inline Library/Formula/libtorrent-rasterbar.rb]: If you're feeling particularly kind, you could bump this to ` sha256 `. Don't worry if you don't want to though, It's not a huge thing or mandatory or such.

File: Library/Formula/libtorrent-rasterbar.rb (hunk 1 above)
Comment (ivanholmes) [inline Library/Formula/libtorrent-rasterbar.rb]: Thanks a lot, nice to understand what's actually going on!

File: Library/Formula/libtorrent-rasterbar.rb (hunk 2 above)
Comment (ivanholmes) [inline Library/Formula/libtorrent-rasterbar.rb]: I *think* I've managed to change it...

Comment (DomT4) [inline Library/Formula/libtorrent-rasterbar.rb]: Yup! Looks good locally. Thanks :smiley_cat:

Comment (ivanholmes) [inline Library/Formula/libtorrent-rasterbar.rb]: Excellent.

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

For future reference the preferred commit message format for simple version updates is `libtorrent-rasterbar 1.0.4`.

**Pull request closed without merging**